### PR TITLE
Added instructions to update Github milestones after release.

### PR DESCRIPTION
**Pull request by Airblader**

### CODE REVIEW

File: release.sh [hunk 1]
```diff
@@ -232,6 +232,10 @@ echo ""
 echo "  cd ${TMPDIR}"
 echo "  sendmail -t < email.txt"
 echo ""
+echo "Update milestones on Github:"
+echo "  Set due date of released version to release date and close the milestone"
```
Comment (stapelberg) [inline release.sh]: Let’s make this a bit more specific: Set due date of ${RELEASE_VERSION} to $(date +'%Y-%m-%d') and close the milestone

File: release.sh [hunk 2]
```diff
@@ -232,6 +232,10 @@ echo ""
 echo "  cd ${TMPDIR}"
 echo "  sendmail -t < email.txt"
 echo ""
+echo "Update milestones on Github:"
```
Comment (stapelberg) [inline release.sh]: nit: GitHub (capital H)

**Comment by Airblader** [comment]
@stapelberg Done

File: release.sh [hunk 3]
```diff
@@ -232,6 +232,10 @@ echo ""
 echo "  cd ${TMPDIR}"
 echo "  sendmail -t < email.txt"
 echo ""
+echo "Update milestones on GitHub:"
+echo "  Set due date of ${RELEASE_VERSION} to $(date + '%Y-$m-%d') and close the milestone"
```
Comment (stapelberg) [inline release.sh]: There’s an extra whitespace between the `+` and the `'`. Please fix.

**Comment by Airblader** [comment]
@stapelberg Fixed now.

**Pull request merged**